src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -92,4 +95,8 @@ public boolean isSecured() { // Checks to see if the game piece is secured
 	}
 
 	public void rumble() {}
+
+	public double getDistance() {
+		return Math.pow(distanceSensor.getAverageVoltage(), -1.2045) * 27.726;
```

Comment:
Where did these numbers (and equation) come from?

Reply:
google apparently it converts it to cm

Reply:
Please document that knowledge with a comment.